omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -42,7 +42,7 @@ protected long getLastModified(HttpServletRequest req) {
             file = getResource(req).getFile();
             log.info("File " + file.getAbsolutePath());
         } catch (IOException e) {
-           log.error("Unable to get the file", e);
```

Comment:
The problem with this change is that it does not log the stack trace. @corneliouzbett was that intentional?

Reply:
No @dkayiwa that wasn't intentional. I did that while debugging to reduce the stack trace while I didn't need it at the moment.